site/docs/master/restic.md

```diff
@@ -84,47 +84,6 @@ You're now ready to use Velero with restic.
 
 ## Back up
 
-1. Run the following for each pod that contains a volume to back up:
```

Comment:
This shouldn't be removed - this is still the way to indicate which volumes the user **wants** to back up.